src/pages/index.tsx

```diff
@@ -112,21 +111,16 @@ const HomeBaseContent = () => {
     );
   }, []);
 
-  if (colorMode === "dark") {
-    return (
-      <>
-        <StarrySky />
-        {mainContent}
-      </>
-    );
-  } else {
     return (
       <>
-        <AuroraBackground />
+        <ThemedImage sources={{
```

Comment:
The current issue has shifted to whether to use more exposed methods to display components or to switch to a different resource for display (such as rewriting it as a GIF) ;s